custom-script/utils.py

```diff
@@ -94,19 +128,7 @@ def parse_model(model, js_output_filename=None):
     return model_javascript
 
 
-def visualize(patch, factor=3.5):
-    fig, ax = plt.subplots(ncols=3, figsize=(22, 7))
-    ax[0].imshow(factor * patch.data["BANDS-S2-L1C"][0][..., [3, 2, 1]].squeeze())
-    ax[0].set_title(f"True color, {patch.timestamp[0]}")
-
-    ax[1].imshow(patch.data_timeless["DEM"].squeeze())
-    ax[1].set_title("DEM")
-
-    ax[2].imshow(patch.mask_timeless["water_label"].squeeze(), vmin=0, vmax=1)
-    ax[2].set_title("water mask")
-
-
-def predict_on_sh(model_script, bbox, size, timestamp, config):
+def predict_on_sh(model_script: str, bbox: BBox, size: Tuple[int], timestamp: list, config: SHConfig) -> np.ndarray:
```

Comment:
the type of `size` is likely wrong, the type of `timestamp` is probably not specific enough

Reply:
done